homeassistant/components/acmeda/sensor.py

```diff
@@ -41,15 +42,15 @@ def async_add_acmeda_sensors():
 class AcmedaBattery(AcmedaBase, SensorEntity):
     """Representation of a Acmeda cover device."""
 
-    device_class = SensorDeviceClass.BATTERY
+    _attr_device_class = SensorDeviceClass.BATTERY
```

Comment:
I'm pretty sure that we shouldn't be overriding the device_class property this way, so I've marked this as a bug fix.